devtools/conda-envs/test_env.yaml

```diff
@@ -49,3 +49,4 @@ dependencies:
   - arsenic
   - click
   - rich
+  - cloudpathlib-all
```

Comment:
Do we want to support all cloudpathlib supported platforms or maybe just `s3`, so far? Just thinking about the conda environments, because if we install `cloudpathlib-all` we would be adding [42 new packages](https://gist.github.com/ijpulidos/a68c38fb5eeea76bf6d329dda8a7d34d) to our current testing and production environments. Whereas, if we only do `cloudpathlib-s3` the addition would be 6 packages, as far as I can see.

I guess we could recommend our users to install the `cloudpathlib-all` if they need to use other cloud platforms.

Reply:
Maybe we do `cloudpathlib-all` for conda-forge package and `cloudpathlib-s3` for CI?

Reply:
We want to use all and if this generates any problems we can change it in the future.